setup.py

```diff
@@ -51,7 +51,7 @@
       data_files=[],
       zip_safe=False,
       install_requires=install_requires,
-      tests_require=['mock'],
+      tests_require=['mock', 'pytroll-schedule', 'pyresample'],
```

Comment:
Do we need these packages for the tests or could (should?) they be mocked?

Reply:
I can mock them if you prefer